assets/styles.css

```diff
@@ -1,3 +1,409 @@
+  border-radius: 4px;
+  display: flex;
+  justify-content: center;
+  min-width: 26px;
+  min-height: 14px;
+  padding: 4px;
+  position: relative;
+  width: auto;
+}
+.hero-figure {
+  width: 90%;
+}
+.hero-figure img {
+  width: 80%;
+  padding-left: 150px;
+  padding-right: 150px;
+  cursor: pointer;
+}
+figcaption {
+  width: 60%;
+  padding-left: 145px;
+}
+figcaption span {
+  margin-right: 7px;
+  color: #666;
+  font-family: nyt-imperial, georgia, "times new roman", times, serif;
+  font-size: 0.875rem;
+  line-height: 1.125rem;
+}
+
+.big-band p {
+  margin-bottom: 0.78125rem;
+  margin-top: 0;
+  font-family: nyt-imperial, georgia, "times new roman", times, serif;
+  font-size: 1.125rem;
+  line-height: 1.5625rem;
+  margin-left: 20px;
+  margin-right: 20px;
+  width: calc(100% - 40px);
+  max-width: 600px;
+  color: #333;
+  display: block;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+}
+footer {
+  padding-bottom: 20px;
+  border-top: 1px solid #ebebeb;
+  padding-top: 9px;
+  margin: 0 auto;
+  width: 1200px;
+  text-align: center;
+  font-size: 11px;
+}
+footer a {
+  padding-inline-start: 20px;
+  text-decoration: none;
+  color: #333;
+}
+.credits {
+  color: #888;
+  letter-spacing: 0.01em;
+  font-size: 0.8125rem;
+  font-family: nyt-imperial, georgia, "times new roman", times, serif;
+}
+.big-band {
+  width: 600px;
+  margin: 0 auto;
+}
+nav {
+  margin: 0 auto;
+  width: 1200px;
+  padding-bottom: 20px;
+  padding-top: 9px;
+  border-bottom: 1px solid #e2e2e2;
+  border-top: 2px solid #e2e2e2;
+  content: "";
+  display: block;
+}
+.nav-header {
+  display: flex;
+  flex-flow: row;
+  height: 45px;
+}
+.go-to-home {
+  margin: 18px 0px 0px auto;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  font-size: 11px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 11px;
+}
+.go-to-home a {
+  color: rgb(86, 123, 149);
+  text-decoration: none;
+}
+.nav-sections {
+  display: flex;
+  flex-flow: row;
+  padding-right: 20px;
+  /* text-decoration: none; */
+  color: #333;
+  width: 1200px;
+  margin: 0 auto;
+}
+.nav-section-column {
+  width: 20%;
+}
+.nav-section-column h3 {
+  color: rgb(51, 51, 51);
+  font-size: 13px;
+  font-weight: 700;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  height: 25px;
+  line-height: 15px;
+  margin: 0px;
+  text-transform: uppercase;
+  width: 150px;
+}
+.nav-section-column ul {
+  list-style: none;
+  margin: 0px;
+  padding: 0px;
+  font: inherit;
+}
+.nav-section-column li a {
+  list-style: none;
+  margin: 0px;
+  padding: 0px;
+  color: rgb(0, 0, 0);
+  display: inline-block;
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  text-decoration: none;
+  text-transform: capitalize;
+  width: 150px;
+  font-size: 14px;
+  font-weight: 500;
+  height: 23px;
+  line-height: 16px;
+}
+.subscribe {
+  border-left: 1px solid rgb(226, 226, 226);
+  /* flex: 1 1 0%; */
```

Comment:
- [ ]  Commented code is not a good practice in clean coding. Please avoid it.

Reply:
Done.